src/composables/multisigTransactions.ts

```diff
@@ -42,8 +43,44 @@ export function useMultisigTransactions({ store }: UseMultisigTransactionsOption
     });
   }
 
-  async function getTransactionByHash(txHash: string) {
-    return fetchJson(`${activeNetwork.value.multisigBackendUrl}/tx/${txHash}`);
+  /**
+ * Get raw proposed transaction from multisig backend
+ * @param {string} txHash
+ * @returns {IRawMultisigTx | null} returns transaction if exists or null
```

Comment:
As we are using TypeScript in my opinion there is no need to use DocBlock here.

Reply:
DocBlock? you mean to remove {IRawMultisigTx | null} from jsdoc?